# Hypothesis Test


##### It is assumed that the mean systolic blood pressure is μ = 120 mm Hg. In the Honolulu Heart Study, a sample of n = 100 people had an average systolic blood pressure of 130.1 mm Hg with a standard deviation of 21.21 mm Hg. Is the group significantly different (with respect to systolic blood pressure!) from the regular population?

- Set up the hypothesis test.
- Write down all the steps followed for setting up the test.
- Calculate the test statistic by hand and also code it in Python. It should be 4.76190. What decision can you make based on this calculated value?

In [1]:
import numpy as np
import pandas as pd
import scipy.stats as stats
import matplotlib.pyplot as plt
import seaborn as sns

%matplotlib inline

### Hypothesis test
- H0: μ= 120; H1: μ!= 120
- confidence level= 95%
- Using a t-score instead of a z-score because we don't have access to the population standard deviation







### Data
1. T = (x – μ) / (σ/√(n))
2. x = sample mean = 130.1
3. μ= population mean = 120
4. σ= sample standard deviation = 21.21
5. n = sample number= 100

In [3]:
sample_mean = 130.1
population_mean = 120
sample_std = 21.21
n = 100

In [11]:
#test static
t_score = (sample_mean - population_mean)/(sample_std/np.sqrt(n))
print("t-score is " + str(t_score))

t_score is 4.761904761904759


Critic value of t from the Student's t distribution table with  = 0.05 and (n - 1) degrees of freedom:



In [12]:
tc = stats.t.ppf(.975, df=99)
print("critical value  is " + str(tc))

tc is 1.9842169515086827


We compare both values:
Since 4.7619(t_score)> 1.9842(tc), we reject the null hypothesis and accept our Ha: μ!= 120